bzt/modules/blazemeter.py

```diff
@@ -1452,10 +1451,17 @@ def check(self):
     def post_process(self):
         if not self.detach and self.router and not self.test_ended:
             self.router.stop_test()
+
         if self.results_url:
             if self.browser_open in ('end', 'both'):
                 open_browser(self.results_url)
 
+        if self.router and self.router.master:
+            full = self.router.master.get_full()
+            for session in full.get('sessions', ()):
+                for error in session.get("errors", ()):
+                    raise TaurusException(to_json(error))
```

Comment:
1. I guess TaurusException is too general, TaurusNetworkError should be better.
2. If there is possibility of  several errors encountering, might be better to put all errors into exception message? What about little formatting? ('message: details\n')